static/js/sefaria/sheetsUtils.js

```diff
@@ -66,4 +66,52 @@ function shouldBeSegmented(ref){
     */
     const categories =  Sefaria.refCategories(ref);
     return !(categories[0] in {"Tanakh": 1, "Talmud": 1});
+}
+    /*
+    Temporarily disables all user interactions (e.g. mouse, touch, keyboard, clipboard, form input) on a given DOM element and its children.
+    */
+export const disableUserInput = (root) => {
+      if (!root) return;
```

Comment:
change to `if (!root) { return; }`. this style prevents issues if the if statement needs to grow beyond one line